src/Http/Routing/WebRouting.php

```diff
@@ -93,6 +94,15 @@ public function wire(): void
             $this->router->delete("/speakers/{speaker}", "destroy")->name("speakers.destroy");
         });
 
+        $this->router->controller(NewsController::class)->group(function (): void {
+            $this->router->get("/news", "index")->name("news");
+            $this->router->get("/news/create", "create")->middleware("auth")->name("news.create");
+            $this->router->post("/news", "store")->name("news.store");
```

Comment:
Please also attach middleware here, as in the store method you are trying to get a user from request

Reply:
done